docs/src/wallet/ledger-live.md

```diff
@@ -0,0 +1,54 @@
+#Ledger Live and Ledger Nano S
+This document describes how to set up a
+[Ledger Nano S hardware wallet](https://shop.ledger.com/products/ledger-nano-s)
+with the [Ledger Live](https://www.ledger.com/ledger-live) software.
+
+**NOTE: While Solana tools are fully integrated with the Ledger Nano S device,
+and the Solana App can be installed on the Nano S using Ledger Live, adding and
+managing wallet accounts currently requires use of our command-line tools.
+Integration with Ledger Live to use Solana wallet accounts on Ledger Live
+will be available in the future.**
+
+Users may use a Ledger Nano S with the Solana command line tools.
+Instructions for how to do that are found [here](../remote-wallet/ledger.md).
```

Comment:
I think it's bad practice to link via the word "here". It means the user has to read the content and can't just skim the links.